components/card/card.css

```diff
@@ -46,3 +46,20 @@
   text-align: right;
   background-color: var(--color-water-10);
 }
+.transform-card {
+  transition: 0.2s ease-out;
+  transform: rotateY(360deg);
+}
+
+.transform-card-back {
+  transition: 0.3s ease-out;
+  transform: rotateY(0deg);
+}
+.card--front-side {
```

Comment:
Not sure but is this supposed to be here?

Reply:
no, i will fix it.